README.md

```diff
@@ -438,6 +438,120 @@ const clientWithNoCache = await Client.create(wallet, {
 })
 ```
 
+## Group chat
+
+![Status](https://img.shields.io/badge/Feature_Status-Pre--Preview-red)
+
+> **Important:**  
+> This feature is in **Pre-Preview** status and ready for you to start experimenting with. We do **not** recommend using Pre-Preview features in production apps. Software in this status will change as we iterate based on feedback.
+
+Use the information in this section to **experiment** with providing group chat in your app.
+
+### Enable group chat for your Client
+
+The first step is to enable group chat for your Client:
+
+```ts
+const creatorClient = await Client.create(yourSigner)
+creatorClient.enableGroupChat()
+```
+
+This enables the following capabilities required to provide group chat:
+
+- The client will be able to create group chats
+- Group chats will be present in `client.conversations.list()`
+- The client will understand GroupChat codecs such as `GroupChatMemberAdded` and `GroupChatTitleChanged`
+
+### Create a group chat
+
+Enable a user to create a group chat using `newGroupConversation` and adding member addresses to it:
```

Comment:
Just a thought as I was editing this - any thoughts on the difference between using `GroupChat...` and `GroupConversation...` in the code? 

Colloquially, I think we call this feature "group chat" - but I think " group conversation" might be more in line with the "conversation" code patterns in XMTP. Any thoughts? 

Whether we go with `GroupChat` or `GroupConversation` - might it be possible to use one pattern throughout? Or are we using the different patterns to indicate a difference? <3

Reply:
The way I've been thinking about it is:

`GroupConversation` is similar to `ConversationV1` or `ConversationV2` from the SDK. It's just a way to send/receive messages.

`GroupChat` is a wrapper around `GroupConversation` that knows about things like group titles, keeping the member list in sync, basically any richer features beyond just sending/receiving messages.

I can see how this could lead to some confusion though. I wonder how we can best clarify it..

Reply:
So good, @nakajima - thank you! I added this section above to help disambiguate: https://github.com/xmtp/xmtp-js/pull/388/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R450